Docker/base-images/entando-postgresql95-base/s2i/bin/assemble

```diff
@@ -31,7 +31,12 @@ configure_mirrors
 if [ -d  $LOCAL_SOURCE_DIR/src/main/db/tar ] && ! [ -d  $LOCAL_SOURCE_DIR/src/main/webapp/protected/databaseBackups/develop/portDataSource ]; then
     ant PG-db-full-update-tar -Dpostgres.bin.path=/opt/rh/rh-postgresql95/root/usr/bin/
 else
-    mvn_command="mvn jetty:run --settings $HOME/.m2/settings.xml $MAVEN_ARGS_APPEND \
+    sed -i  '/\<webApp>/a <webInfIncludeJarPattern>nothing</webInfIncludeJarPattern>' pom.xml
```

Comment:
A comment on the purpose and why of this would be good.

Reply:
Fair enough. Done.